LiteCore/Storage/SQLiteKeyStore.cc

```diff
@@ -584,7 +584,7 @@ namespace litecore {
     expiration_t SQLiteKeyStore::nextExpiration() {
         expiration_t next = expiration_t::None;
         if (mayHaveExpiration()) {
-            auto &stmt = compileCached("SELECT min(expiration) FROM kv_@");
+            auto &stmt = compileCached("SELECT min(expiration) FROM kv_@ where expiration NOT NULL");
```

Comment:
Good catch -- the index definition says `NOT NULL` so the query has to include it too.

Reply:
This issue has been found in a separate ticket. It's fixed in release/lithium, 1e2cbb4a1